oauth2_http/pom.xml

```diff
@@ -182,13 +144,6 @@
           </excludes>
           <reportNameSuffix>sponge_log</reportNameSuffix>
         </configuration>
-        <dependencies>
-          <dependency>
-            <groupId>org.apache.maven.surefire</groupId>
-            <artifactId>surefire-junit47</artifactId>
-            <version>3.0.0-M5</version>
-          </dependency>
-        </dependencies>
       </plugin>
```

Comment:
A minimum of junit 5.8.1 is required for tests to run with native image compilation([reference](https://graalvm.github.io/native-build-tools/latest/maven-plugin.html#testing-support)) and this dependency prevents tests from running in junit 5. Removing it resolves the following error:
```
Execution test-native of goal org.graalvm.buildtools:native-maven-plugin:0.9.23:test failed: Test configuration file wasn't found.
```

According to the [maven documentation](https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit.html#provider-selection), the provider is automatically selected based on the junit version present in the project dependencies.